pulley/src/imms.rs

```diff
@@ -0,0 +1,28 @@
+//! Immediates.
+
+/// A PC-relative offset.
+///
+/// This is relative to the start of this offset's containing instruction.
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct PcRelOffset(i32);
+
+#[cfg(feature = "arbitrary")]
+impl<'a> arbitrary::Arbitrary<'a> for PcRelOffset {
+    fn arbitrary(u: &mut arbitrary::Unstructured<'a>) -> arbitrary::Result<Self> {
+        Ok(Self(u.arbitrary()?))
+    }
+}
```

Comment:
This seems a bit dangerous, should this perhaps be omitted? Or maybe it always generates 0 for the time being?

(seems like the "real fix" is a higher-level generator that generates semantically correct offsets)

Reply:
Changed this to always choose zero.

Either way, whoever is working with the arbitrary ops is going to have to clean them up somehow if they are interpreting them. In the existing fuzz target for the interpreter we simply filter out instructions that we can't safely interpret, such as branches.